mesonbuild/interpreter.py

```diff
@@ -870,15 +870,23 @@ class JarHolder(BuildTargetHolder):
     def __init__(self, target, interp):
         super().__init__(target, interp)
 
-class CustomTargetIndexHolder(InterpreterObject, ObjectHolder):
-    def __init__(self, object_to_hold):
-        InterpreterObject.__init__(self)
-        ObjectHolder.__init__(self, object_to_hold)
+class CustomTargetIndexHolder(TargetHolder):
+    def __init__(self, target, interp):
+        super().__init__(target, interp)
+        self.methods.update({'full_path': self.full_path_method,
+                            })
```

Comment:
__[Flake8]__ closing bracket does not match visual indentation ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6417#issue-5026023))

<details>

| Rule |
| - |
| E124 |

</details>
<sub>posted by Sider</sub>